apps/i18n/common/en_us.json

```diff
@@ -45,6 +45,8 @@
   "courses": "Courses",
   "coursesCardAction": "Find a course",
   "coursesCardDescription": "Find a new course to keep challenging yourself.",
+  "coursesHeadingText": "Courses",
```

Comment:
Courses is already here. Is it better to have these strings more specific to the place they are used or to re-use the same string if it's in multiple places?